index.js

```diff
@@ -738,6 +738,11 @@
     return function(_) { return x; };
   };
 
+  //  Function$prototype$equals :: Function ~> Function -> Boolean
+  var Function$prototype$equals = function(other) {
+    return other === this;
+  };
+
```

Comment:
Having removed `Object(x) === Object(y)` from `equals` we need to define `Function#fantasy-land/equals`.